tests/test_dse.py

```diff
@@ -2565,11 +2572,11 @@ def test_fullopt(self):
 
         # Check expected opcount/oi
         assert summary[('section1', None)].ops == 90
-        assert np.isclose(summary[('section1', None)].oi, 1.511, atol=0.001)
+        assert np.isclose(summary[('section1', None)].oi, 2.031, atol=0.001)
```

Comment:
? Why?

Reply:
byproduct of a more accurate dspace calculation

Reply:
seems way.. more accurate..